movie.sh

```diff
@@ -1,58 +1,68 @@
 #!bin/bash
 
-function add_style()
-{
+mkdir tmp #Making a folder to dump files
```

Comment:
I think you missed this again? This directory is not needed. Read this http://www.tldp.org/LDP/Linux-Filesystem-Hierarchy/html/tmp.html